web/package.json

```diff
@@ -16,13 +16,16 @@
     "@types/react-router-dom": "^5.3.3",
     "axios": "^1.6.7",
     "firebase": "^10.11.0",
+    "poppins": "^0.1.1",
+    "poppins-font": "^1.0.1",
```

Comment:
The Poppins font doesn't work when you try `className=font-poppins`, so this is probably the issue. That being said, the Work sans font and Lora font work fine.

Reply:
fixed